source/includes/_connected-apps.md

```diff
@@ -1,84 +1,139 @@
+Later, you will receive a different set of client credentials for our live environment.
+
+Your technical team will need to provide TransferWise with a *redirect URL* for the account linking flow.
+This is the URL controlled by your system that we will forward users to after successfully granting your application
+access to TransferWise accounts.
+Specifying this URL explicitly makes the integration more secure.
+We will only use redirect URLs registered with us when redirecting our users to your system.
+
+<a href="https://auth0.com/docs/protocols/oauth2" target="_blank" rel="noopener noreferrer">This article about the OAuth 2.0 framework</a>
+is a great way to refresh your knowledge about the protocol itself.
 
-Once our partnership begins, we’ll send you API access credentials for the sandbox environment consisting of a Client ID and a Client Secret.
-The credentials are needed to complete the authorisation_code grant type of OAuth 2.0 through which the customer will allow your application to be able to act on their behalf.
-We also need *redirect_url* from your technical team so that we can limit our callbacks to only that URL, this is URL that we will forward users to after successfully granting your application access to their TransferWise account. Specifying this explicitly makes the integration more secure.
-<a href="https://auth0.com/docs/protocols/oauth2" target="_blank" rel="noopener noreferrer">This article about OAuth 2.0 framework</a> is a great way to refresh your knowledge about the protocol itself.
 
-### SANDBOX and LIVE environments
+## Sandbox and live environments
 
-* Sandbox API is located at https://api.sandbox.transferwise.tech
-* LIVE API is located at https://api.transferwise.com
+* Our sandbox (test) API is located at https://api.sandbox.transferwise.tech
 
-## Get user authorization
+* Our live (production) API is located at https://api.transferwise.com
 
-At a high level there are three steps to gaining access to an existing TransferWise account.
-<ol>
-  <li>Your app redirects the user to TransferWise authorization webpage, which prompts customers to login.<br/>
-  </li>
-  <li>The user logs in to TransferWise.</li>
-  <li> The user agrees to provide access, the TransferWise authorization page then redirects user back to your preconfigured url, including a code you can use to geneate user tokens. e.g.
-  `
-  https://www.yourapp.com/?code=[CODE]
-  `
-  </li>
-</ol>
+
+## User authorization
+
+At a high level, there are five steps to gaining access to an existing TransferWise account:
+
+1. Your application redirects the user to the TransferWise authorization page, which prompts them to log in.
+2. The user logs in to TransferWise.
+3. The user agrees to give your application access to one of their TransferWise profiles.
+4. TransferWise redirects the user back to your preconfigured redirect URL, with an authorization code and selected profile ID appended as URL query parameters.
```

Comment:
wonder if we should explain what's a profile(or link to a page where it's explained)

Reply:
nvm, it's explained below